presto-docs/src/main/sphinx/release/release-337.rst

```diff
@@ -0,0 +1,55 @@
+=========================
+Release 337 (XX Jun 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect results for inequality join involving ``NaN``. (:issue:`4120`)
+* Fix peak non-revocable memory metric in event listener. (:issue:`4096`
+* Improve performance of queries involving comparisons between ``DOUBLE`` or ``REAL`` values and integer types. (:issue:`3533`)
+* Add peak non-revocable memory metric to query stats. (:issue:`4096`)
+* Add support for variable-precision ``TIMESTAMP WITH TIME ZONE`` type (:issue:`3947`)
+* Add support for ``IN`` predicate with subqueries in outer join condition. (:issue:`4151`)
+* Add support for quantified comparisons (e.g., ``> ALL (...)``) in aggregation queries. (:issue:`4128`)
+* Add Druid connector. (:issue:`3522`)
```

Comment:
Are there docs for this? Can we throw together some basic docs so it shows up in the connector list?